endpoints/setuid.go

```diff
@@ -73,6 +75,19 @@ func NewSetUIDEndpoint(cfg config.HostCookie, syncersByBidder map[string]usersyn
 			return
 		}
 
+		accountID := query.Get("account")
+		if accountID == "" {
+			accountID = metrics.PublisherUnknown
+		}
+		_, fetchErrs := accountService.GetAccount(context.Background(), cfg, accountsFetcher, accountID)
+		if len(fetchErrs) > 0 {
+			w.WriteHeader(http.StatusBadRequest)
+			w.Write([]byte(combineErrors(fetchErrs).Error()))
+			metricsEngine.RecordSetUid(metrics.SetUidBadRequest)
```

Comment:
Nitpick question: should we capitalize the `id` in `RecordSetUid` and `SetUidBadRequest` just to be consistent with the function name `NewSetUIDEndpoint`?

Reply:
We should but I think that's outside of the scope of this PR, especially since this issue has existed for quite some time and because there are a lot of occurrences. We can open another PR to fix this.